index.html

```diff
@@ -554,6 +554,775 @@ <h2>OpenID Connect</h2>
             </ol>
           </section>
         </section>
+
+        <!-- Controller documents start -->
+        <section>
+        <h3>Controller Documents</h3>
+
+        <p>
+A <a>controller document</a> is a set of data that specifies one or more
+relationships between a <a>controller</a> and a set of data, such as a set of
+public cryptographic keys. The <a>controller document</a> SHOULD
+contain <a>verification relationships</a> that explicitly permit the use of
+certain <a>verification methods</a> for specific purposes.
+        </p>
+
+        <div class="issue">Add examples of common Controller documents, such as
+controller documents published on a ledger-based registry, or on a mutable medium in
+combination with an integrity protection mechanism such as Hashlinks.
+        </div>
+
+        <section>
+          <h2>Verification Methods</h2>
+          <p>
+A <a>controller document</a> can express <a>verification methods</a>, such as
+cryptographic <a>public keys</a>, which can be used to <a>authenticate</a> or
+authorize interactions with the <a>controller</a> or associated parties. For
+example, a cryptographic <a>public key</a> can be used as a <a>verification
+method</a> with respect to a digital signature; in such usage, it verifies that
+the signer could use the associated cryptographic private key. <a>Verification
+methods</a> might take many parameters. An example of this is a set of five
+cryptographic keys from which any three are required to contribute to a
+cryptographic threshold signature.
+          </p>
+
+          <dl>
+            <dt><dfn class="lint-ignore">verificationMethod</dfn></dt>
+            <dd>
+              <p>
+The `verificationMethod` property is OPTIONAL. If present, the value
+MUST be a <a data-cite="INFRA#ordered-set">set</a> of <a>verification
+methods</a>, where each <a>verification method</a> is expressed using a <a
+data-cite="INFRA#ordered-map">map</a>. The <a>verification method</a> <a
+data-cite="INFRA#ordered-map">map</a> MUST include the `id`,
+`type`, `controller`, and specific verification material
+properties that are determined by the value of `type` and are defined
+in <a href="#verification-material"></a>. A <a>verification method</a> MAY
+include additional properties. <a>Verification methods</a> SHOULD be registered
+in the Data Integrity Specification Registries [TBD - DIS-REGISTRIES].
+              </p>
+              <p class="note">
+The `verificationMethod` property is REQUIRED for proofs, unlike controller
+documents, for which it is optional. See section <a href="#proofs"></a>.
```

Comment:
this is data integrity specific and needs to be removed or adjusted for JOSE / COSE.